tasks/us/bls.py

```diff
@@ -256,7 +257,7 @@ def populate(self):
                 END)::Numeric'''.format(naics_code=naics_code,
                                         colname=colname))
         insert = '''INSERT INTO {output} ({colnames})
-                    SELECT area_fips, {select_colnames}
+                    SELECT area_fips, area_fips, {select_colnames}
```

Comment:
Maybe add to the SELECT something like `AS area_fipssl` or `AS area_fipssc` to improve readability?